src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -307,8 +304,6 @@ public static void TCP_ServiceCertExpired_Throw_MessageSecurityException()
             {
                 Assert.True(false, string.Format("Expected type SecurityNegotiationException, Actual: {0}", exceptionType));
             }
-            string exceptionMessage = exception.Message;
```

Comment:
Reviewers: in NET Native, this message is different due to the way resources work.  We have gradually been removing these exception message verification asserts as we find them.